src/net_processing.h

```diff
@@ -94,7 +94,8 @@ class PeerManager : public CValidationInterface, public NetEventsInterface
     virtual void PushInventory(NodeId nodeid, const CInv& inv) = 0;
 
     /** Relay inventories to all peers */
-    virtual void RelayInv(CInv &inv, const int minProtoVersion = MIN_PEER_PROTO_VERSION) = 0;
+    virtual void RelayInv(CInv &inv) = 0;
```

Comment:
Peers with a version lower than this are disconnected asap:
https://github.com/dashpay/dash/blob/develop/src/version.h#L19-L20
https://github.com/dashpay/dash/blob/develop/src/net_processing.cpp#L3447-L3452

Side note: this made me wonder what versions are used in `RelayInv()` calls and we might want to change this line https://github.com/dashpay/dash/blob/develop/src/net_processing.cpp#L3394 so that `PostProcessMessage()` would use the fast variant too:
```diff
-        RelayInv(result.m_inventory.value(), MIN_PEER_PROTO_VERSION);
+        RelayInv(result.m_inventory.value());
```
Can be done here or in a follow-up PR.